lib/bootstrap.js

```diff
@@ -1,11 +1,20 @@
-const webpackCli = require('./webpack-cli');
+const WebpackCLI = require('./webpack-cli');
 const { core, commands } = require('./utils/cli-flags');
 const cmdArgs = require('command-line-args');
 
 require('./utils/process-log');
 
 const isFlagPresent = (args, flag) => args.find(arg => [flag, `--${flag}`].includes(arg));
-const stripDashedFlags = (args, cmd) => args.slice(2).filter(arg => ~arg.indexOf('--') && arg !== cmd.name && arg !== cmd.alias);
+const isArgCommandName = (arg, cmd) => arg === cmd.name || arg === cmd.alias;
+const stripDashedFlags = (args, cmd) => args.filter(arg => ~arg.indexOf('--') && !isArgCommandName(arg, cmd));
```

Comment:
```suggestion
const stripDashedFlags = (args, cmd) => args.filter(arg => arg.includes('--') && !isArgCommandName(arg, cmd));
```

Reply:
Actually I think the logic got messed up and I just followed along with it :slightly_frowning_face:

It should be `!arg.includes('--')`, because we want to get rid of all the arguments with `--` in this helper, correct?